SB.js

```diff
@@ -184,8 +184,8 @@ function resetConfig() {
     SB.config = SB.defaults;
 };
 
-function convertJSON() {
-    Object.keys(SB.defaults).forEach(key => {
+function decodeConfig() {
+    Object.keys(SB.localConfig).forEach(key => {
```

Comment:
Oh shoot, how was this ever working?

Reply:
If the key was used in SB.defaults it would still get decoded

Reply:
Isn't the sponsorTimes in the defaults? Doesn't that mean that this shouldn't change the behavior?

Reply:
It is in the defaults however there maybe something thats not.

Reply:
I have been unable to get the error after this change maybe it only happens when the video has no times